### PR TITLE
Update CI configs to v0.4.7

### PR DESCRIPTION
**Pull request by pionbot**
Update lint scripts and CI configs.

### CODE REVIEW

**Comment by scorpionknifes** [comment]
I am taking this one :)

File: jsep.go [hunk 1]
```diff
@@ -195,6 +195,9 @@ func (d *MediaDescription) WithExtMap(e ExtMap) *MediaDescription {
 
 // WithTransportCCExtMap adds an extmap to the media description
 func (d *MediaDescription) WithTransportCCExtMap() *MediaDescription {
+	extMapURI := map[int]string{
+		ExtMapValueTransportCC: "http://www.ietf.org/id/draft-holmer-rmcat-transport-wide-cc-extensions-01",
+	}
```
Comment (scorpionknifes) [inline jsep.go:200]: Make extMapURI a non-global variable to pass "no global variable". extMapURI seems to be not exported or used anywhere?

**Review by scorpionknifes — COMMENTED** [review]
Do I need to add more tests to maintain test coverage?

**Review by Sean-Der — APPROVED** [review]

**Comment by Sean-Der** [comment]
Amazing job again @scorpionknifes you are burning through these.

I appreciate it so much :)

**Pull request merged**